mondrian/src/main/java/mondrian/olap/Mondrian.xml

```diff
@@ -966,7 +966,7 @@ Revision is $Id$
         <Attribute name="datatype" required="false">
             <Doc>
                 The datatype of this measure:
-                String, Numeric, Integer, Boolean, Date, Time or Timestamp.
+                String, Numeric, Integer.
 
```

Comment:
Boolean, Date, Time and Timestamp are intended to be supported as measure datatypes, if I'm not mistaken. 
http://mondrian.pentaho.com/documentation/schema.php#Measures

Tried a quick test with Boolean using count / distinct-count aggregators, seemed to work fine.

I think this bug should probably be to make PSW allow those data types.